src/components/common/modal/DefaultModal.tsx

```
@@ -0,0 +1,56 @@
+}: DefaultModalProps) {
+  return (
+    <Modal
+      transparent
+      animationType="slide"
+      visible={visible}
+      onRequestClose={onCancel}
+    >
+      <S.Backdrop>
+        <S.ModalContainer>
+          <S.TextBox>
+            <Typography font="bold20" color="defaultBlack" children={title} />
```

Biome (2.4.6)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)